pkg/action/executor/resize_container.go

```diff
@@ -248,6 +248,19 @@ func (r *ContainerResizer) resizeControllerContainer(pod *k8sapi.Pod, parentKind
 	glog.V(3).Infof("resizeContainer [%s]: got lock for parent[%s]", id, parentName)
 	helper.KeepRenewLock()
 
+	// Performs operations for actions to be non-disruptive (when the pod is the only one for the controller)
+	// NOTE: It doesn't support the case of the pod associated to Deployment in version lower than 1.6.0.
+	//       In such case, the action execution will fail.
+	contKind, contName, err := util.GetPodGrandInfo(r.kubeClient, pod)
```

Comment:
`SchedulerHelper` has pod's parent name and kind, its grandParent info may be get through on its parent's info. (one less API call)

Reply:
Discussed with Songbin. I will keep it as getting the parent's info doesn't invoke additional API call.